src/index.js

```diff
@@ -1,24 +1,20 @@
+import addScore from './modules/addScores.js';
+import getScores from './modules/getScores.js';
 import './style.css';
-import addLi from './modules/add-li.js';
 
-const leaderboardData = {
-  result: [
-    {
-      user: 'John Doe',
-      score: 42,
-    },
-    {
-      user: 'Peter Parker',
-      score: 35,
-    },
-    {
-      user: 'Wonder Woman',
-      score: 50,
-    },
-  ],
-};
+const scoresList = document.querySelector('.scores-list');
+const submitButton = document.querySelector('#submit-score');
+const refreshButton = document.querySelector('#refresh');
+const message = document.querySelector('.message');
 
-leaderboardData.result.forEach((element) => {
-  const text = `${element.user}: ${element.score}`;
-  addLi(text);
+refreshButton.addEventListener('click', (e) => {
+  e.preventDefault();
+  getScores(scoresList);
+});
```

Comment:
As the first code reviewer said 

> It would be a great addition if you also fetch this data when loading the page, not only after clicking on the refresh button. This would display the score list right after loading the elements

- [ ] I tried to use your app but it didn't work. Please review the last changes and correct the issues. 👇🏽 

https://user-images.githubusercontent.com/58642949/197362304-8be5a975-7c8d-4cf2-98eb-fd2d96f4058a.mp4